.env.sample

```diff
@@ -0,0 +1 @@
+REACT_APP_LANG=en
```

Comment:
Supposed to be `en`, I told you to look it up:
https://github.com/jainaayush01/easya-website/blob/patch1/src/constants/Translations/Translations.js#L21